src/openms/source/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderIdentificationAlgorithm.cpp

```diff
@@ -139,7 +139,7 @@ namespace OpenMS
     // exclude some redundant/uninformative scores:
     // @TODO: intensity bias introduced by "peak_apices_sum"?
     // names of scores to use as SVM features
-    String score_metavalues = "peak_apices_sum,var_xcorr_coelution,var_xcorr_shape,var_library_sangle,var_intensity_score,sn_ratio,var_log_sn_score,var_elution_model_fit_score,xx_lda_prelim_score,var_isotope_correlation_score,var_isotope_overlap_score,var_massdev_score,main_var_xx_swath_prelim_score";
+    String score_metavalues = "peak_apices_sum,var_xcorr_coelution,var_xcorr_shape,var_library_sangle,var_intensity_score,sn_ratio,var_log_sn_score,var_elution_model_fit_score,xx_lda_prelim_score,var_ms1_isotope_correlation_score,var_ms1_isotope_overlap_score,var_massdev_score,main_var_xx_swath_prelim_score";
```

Comment:
What about the RT score in FFID? 
iRT in the library can be set as normal RT in seconds (-Library:retentionTimeInterpretation 'seconds') and setting the rt normalization factor to the length of the gradient used in seconds (-Scoring:rt_normalization_factor 1200.0).

Reply:
What exactly does it measure? There was a comment slightly above that says the RT scores are a bit unreliable.

Reply:
@oliveralka ping

Reply:
The rt_score basically the difference to the retention time in the library. The smaller the difference the better. This comes in handy if the chromatography is similar / measured at the same column/instrument. It depends a little bit on how big your extraction window is. You have an extraction window of 600s seconds and find two similar peak groups one at 1300s (1) and the other one at 1600s rt (2) - your peak group in the library is at 1580s <- so in terms of the retention time score peak group 2 would be better off with a difference of 20s instead of 280s.

I am not sure about the unreliability statement. Chromatography always is dependent on the experimental setup!

Reply:
I guess you have a really small extraction window in the FFMID, so the score is probably not needed here.

Reply:
Hmm also, if transitions are "transferred" to other maps it depends on how good the alignment of the maps worked I guess.
The extraction windows are not so small since it sometimes tries to pick up same IDs that elude multiple times and because of the aforementioned transfer.

Reply:
I guess this is why usually TRIC or DIAlignR are used after OpenSWATH, to realign "mismatched" peak groups based on the chromatograms.

Reply:
The unreliability (as far as FFId is concerned) comes from the fact that the library RT is only an estimate (for "external" IDs). If we knew the exact RT where to expect a feature, we wouldn't need any sophisticated scoring to choose between feature candidates.